tools/CoqMakefile.in

```diff
@@ -618,7 +618,7 @@ install-extra::
 
 META: $(METAFILE)
```

Comment:
Question (possibly not for this PR): should this be
```suggestion
META: "$(METAFILE)"
```
?

Is `METAFILE` ever empty?